assets/scss/brandings-dark-background--php-used.scss

```diff
@@ -0,0 +1,71 @@
+/*******
+	This file creates the styles for the dark backgrounds which can be selected in the WordPress Palette.
+	There are 5 greyscale colours and 4 "generic" colours.
+	The generic colours vary from site to site.
+	They might be dark or they might not be, so we cannot know whether to apply the "dark-text" (etc) colours or not.
+	This file creates the CSS for a dark background, but it uses the class "has-replace_replace-background-color" (which isn't used).
+	We use this CSS in PHP (colour-branding.php) to apply the style should the colour chosen be calculated to be dark.
+	So whilst this CSS is not directly used by anything, it is copied by the PHP, amended and added where needed.
+*******/
+
+@mixin hale-block-inside-block($paletteColour, $colour, $hoverColour: "") {
+	// @include hale-block-inside-block($colour, $hoverColour)
```

Comment:
Is this ment to be commented out? if so put some info why

Reply:
Removed this line - not needed.